.modernizrrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "minify": true,
+  "options": [
+    "setClasses"
+  ],
+  "feature-detects": [
```

Comment:
Does this make it so we are only importing these two bits of code vs the whole set of feature detects?